templates/web/pages/about.tmpl

```diff
@@ -0,0 +1,142 @@
+
+				<p> <br> If this sort of stuff sounds like your thing, stay tuned for more updates!
+					We’ll be looking for new people to help us throw another successful hackathon in
+					2018! As always - if you have any questions, feel free to hit us up at team@boilermake.org! </p>
+			</div>
+		</div>
+	</div>
+</section>
+
+<section class="section">
+	<h1 class="title is-1 is-spaced has-text-centered">History</h1>
+	<div class="columns is-mobile is-vcentered content">
+		<div class="column">
+			<img src='{{static_path "history.jpg"}}' alt="">
+		</div>
+
+		<div class="column">
+			<h2 class="subtitle">BoilerMake I: February 8-10th, 2014</h2>
+			<p>The first BoilerMake started as a group of students involved with
+				Purdue Hackers created BoilerMake as an organization under the
+				Undergraduate Student Board. The goal was to get students involved and
+				throw a hackathon where everyone could come together and create. It
+				was created to be a great way for students to interact informally with
+				companies and proving their technical skills through development.
+				There were around 400 attendees from 18 schools and about 30 companies
+				all within the same space of Purdue’s gym.</p>
+		</div>
+
+
+	</div>
+
+	<div class="columns is-mobile is-vcentered content">
+		<div class="column">
+			<h2 class="subtitle">BoilerMake II: Oct 17-19th, 2014</h2>
+			<p>BM II was the year of the BoilerMake Badges - custom Arduino boards for hackers to have fun with.
+				Our venue changed to the Lambert Fieldhouse to accommodate the 500 hackers. We kept this
+				number as the maximum to ensure that all the hackers and sponsors could fit into the same
+				large room to interact and develop in.</p>
+		</div>
+
+		<div class="column">
+			<img src='{{static_path "about_bm_1.jpg"}}' alt="">
+		</div>
+
+	</div>
+
+	<div class="columns is-mobile is-vcentered content">
+
+		<div class="column">
+			<img src='{{static_path "about_bm_2.jpg"}}' alt="">
+		</div>
+
+		<div class="column">
+			<h2 class="subtitle">BoilerMake III: Oct 16-18th, 2015</h2>
+			<p>BM III was once again located in Lambert Fieldhouse. This year,
+				we provided arcade games, bean bags, and opportunities for hackers
+				to enjoy their full hackathon experience. We also provided a diverse
+				hardware lab for students to experiment and build with a variety
+				of technologies.</p>
+		</div>
+
+	</div>
+
+	<div class="columns is-mobile is-vcentered content">
+		<div class="column">
+			<h2 class="subtitle">BoilerMake IV: January 20-22nd, 2017</h2>
+			<p>BM IV followed a space adventure theme as hackers became astronauts traveling through the stars. This event was once again held in the Cordova Recreational Sports Center among the basketball courts. 500 students came together once again to
+				create under space themed decorations, arcade games, interactive lego boards.</p>
+		</div>
+
+		<div class="column">
+			<img src='{{static_path "about_boileriv_1.jpg"}}' alt="">
```

Comment:
Can we standardize these image names a little? Above they're `about_bm_1.jpg` and here they're `about_boileriv_1.jpg`